heat/core/dndarray.py

```diff
@@ -166,6 +166,28 @@ def split(self):
         """
         return self.__split
 
+    @property
+    def stride(self):
+        """
+        Returns
+        -------
+        tuple of ints: steps in each dimension when traversing a tensor.
+        torch-like usage: self.stride()
+        """
+        return self.__array.stride
+
+    @property
+    def strides(self):
+        """
+        Returns
+        -------
+        tuple of ints: bytes to step in each dimension when traversing a tensor.
+        numpy-like usage: self.strides
+        """
+        stride = np.array(self._DNDarray__array.stride())
```

Comment:
There is even no need to return a array (Numpy or PyTorch) at all as stride() already returns a tuple

Reply:
@krajsek this is the numpy-like version of stride (output in bytes), I need to be able to multiply it by the element size.  

@coquelin77 if you expect problems I'll use list comprehension instead of numpy. Must we be wary of every numpy call then, or only when dealing with large arrays?

Reply:
> o.k., I see
> 
> @krajsek this is the numpy-like version of stride (output in bytes), I need to be able to multiply it by the element size.
> 
> @coquelin77 if you expect problems I'll use list comprehension instead of numpy. Must we be wary of every numpy call then, or only when dealing with large arrays?

Reply:
yes, we should always be wary of using numpy. we should aim to use primarily pytorch functions. Also, if i dont misunderstand, wrapping this in a numpy array doesnt really get us anything since its already a torch tensor

Reply:
Hi Daniel.
This:
`self._DNDarray__array.stride()`
is a tuple. In order to provide numpy-like information (output in bytes), this tuple has to be multiplied by the element size of the tensor (scalar, bytes). I need to change the tuple into something that can be multiplied by a scalar. I'll use list comprehension and be done with it.